apps/wallet-mobile/src/components/BottomSheet/BottomSheet.tsx

```diff
@@ -71,31 +71,31 @@ export const BottomSheet = React.forwardRef<BottomSheetRef, BottomSheetProps>(
 
     return (
       <>
```

Comment:
this won't be part of the modal, it will make this screen part of the current screen, so it won't take over the 100% of the UI.

Reply:
it will take 100% of the nearest parent

Reply:
Yes, one possibility could be to implement a provider at the root of the app and manage the dialog system through contexts.

As the current implementation is adapted to the changes of this PR, one possibility would be to create a specific ticket for that improvement.

Thanks!

Reply:
It would be interesting to create a specific ticket for that since the entire swap feature would have to be refactored again.